.github/workflows/stop-runner.yml

```diff
@@ -19,17 +27,28 @@ on:
         required: true
       AWS_REGION:
         required: true
+      AWS_ACCOUNT_ID:
+        required: true
 
 jobs:
   stop-runner:
     runs-on: ubuntu-latest
     steps:
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1-node16
+      - name: Checkout 10ds actions repo
+        uses: actions/checkout@v4
+        with:
+          repository: PMO-Data-Science/10ds-core-github-actions
+          ref: refs/heads/main
+          token: ${{ secrets.AWS_GITHUBRUNNER_PAT }}
+      - name: AWS Authentication
+        uses: ./.github/actions/authenticate_aws
```

Comment:
I'm unsure about if this is needed to shut down the runners

Reply:
It looks to me that it is needed by the proceeding step, as it runs on a github shared runner and so needs the creds to access the AWS environment. I'm not sure if this step would work with `USE_OIDC=true` in its current state though, as it will use the new deployment roles which don't have the IAM permissions to terminate the ephemeral runners

Reply:
You're right, it wouldn't work currently. It previously wasn't in this stage hence the question, but my assumption is we don't specify it it'll revert to the instance profile of the roles. I'm updating the permissions currently to include this as well as RunInstances!